_maps/yogstation/map_files/generic/CentCom.dmm

```diff
@@ -18397,10 +18903,22 @@
 	opacity = 0
 	},
 /area/bluespace_locker)
+"XX" = (
+/obj/mecha/combat/gygax,
+/turf/open/floor/circuit,
+/area/centcom/control)
 "Ya" = (
 /obj/structure/table/wood,
 /turf/open/floor/wood,
 /area/centcom/holding)
+"Yb" = (
+/obj/structure/table/reinforced,
+/obj/item/gun/energy/laser,
+/obj/item/gun/energy/laser/instakill/red,
```

Comment:
You thought you were going to have fun trying out unique yet powerful weapons on others but its me instakill rifle!!

Reply:
https://cdn.discordapp.com/attachments/553082138866614272/578432346294976522/unknown.png